GitSearch/Application/RepoInfo/RepoInfoCell.swift

```diff
@@ -10,11 +10,80 @@ import UIKit
+        label.font = UIFont.systemFont(ofSize: 13)
+        label.textColor = UIColor(red: 100/255, green: 100/255, blue: 100/255, alpha: 1.0)
+        return label
+    }()
+    
+    let propertyValue: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.boldSystemFont(ofSize: 20)
+        label.numberOfLines = 0
+        return label
+    }()
+    
+    let separator: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = UIColor(red: 225/255, green: 225/255, blue: 225/255, alpha: 1.0)
+        return view
+    }()
+    
+    func removeSeparator() {
+        separator.removeFromSuperview()
+    }
+    
+    func setupSubviews() {
+        contentView.addSubview(title)
+        contentView.addSubview(propertyValue)
+        contentView.addSubview(separator)
+        
+        let borderSize: CGFloat = 25
+        let borderSizeShift: CGFloat = 5
+        let separatorHeight: CGFloat = 1
+
+        let constraints = [
+            title.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 16),
+            title.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: borderSize + borderSizeShift),
+            title.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -borderSize*2),
+
+            propertyValue.topAnchor.constraint(equalTo: title.bottomAnchor, constant: 5),
+            propertyValue.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: borderSize),
+            propertyValue.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -borderSize*2),
+            propertyValue.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -separatorHeight),
+
+            separator.topAnchor.constraint(equalTo: propertyValue.bottomAnchor, constant: 0),
+            separator.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: borderSize - borderSizeShift),
+            separator.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -borderSize*2),
+            separator.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: 0),
+        ]
+        
+        NSLayoutConstraint.activate(constraints)
+    }
+    
+    func textView(_ textView: UITextView, shouldInteractWith URL: URL, in characterRange: NSRange, interaction: UITextItemInteraction) -> Bool {
+        UIApplication.shared.open(URL)
+        return false
+    }
+
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupSubviews()
     }
     
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 }
+
+extension UILabel {
+    func correctFontSize() {
```

Comment:
Why to have this? It creates inconsistencies in design.

Reply:
in progress, guess need to add different cell type